components/Navbar.js

```diff
@@ -43,7 +43,7 @@ export default function Navbar() {
             href={QUICKET_LINK}
           >
             {" "}
-            Register
+            Get Ticket
```

Comment:
@basiima does this mean we standardizing as `get ticket` than `buy ticket` ?

I feel `get ticket` is better.
This should reflect also on the navbar button

Reply:
Yes

Reply:
Awesome